boards/xtensa/esp32s3/esp32s3-meadow/Kconfig

```diff
@@ -15,6 +15,10 @@ config ESP32S3_STORAGE_MTD_OFFSET
 config ESP32S3_STORAGE_MTD_SIZE
 	hex "Storage MTD size in SPI Flash"
 	default 0x100000
```

Comment:
is it intentional to use a different order of default and range? why?

Reply:
No. I probably pasted it wrong, but as the order is not important, the tests didn't fail. I will make it consistent among the devices...

Reply:
Fixed!